### PR TITLE
Update Next with Master Changes

### PR DESCRIPTION
**Pull request by awharn**
**What It Does**

Updates `next` with recent changes on `master` that resolve additional deprecations and file conflicts.
Fixes a situation where running tests locally on MacOS results in broken tests due to keychain operations.

**How to Test**

**Review Checklist**
I certify that I have:
- [x] tested my changes
- [ ] added/updated automated tests
- [x] updated the changelog
- [x] followed the [contribution guidelines](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md)

**Additional Comments**

### CODE REVIEW

**Review by gejohnston — APPROVED** [review]

File: __tests__/__packages__/cli-test-utils/src/TestUtils.ts [hunk 1]
```diff
@@ -135,6 +135,7 @@ export function mockHandlerParameters(params: PartialHandlerParameters): IHandle
             ...params.arguments || {}
         },
         positionals: params.positionals || [],
+        // eslint-disable-next-line deprecation/deprecation
         profiles: params.profiles || new CommandProfiles(new Map()),
```
Comment (t1m0thyj) [inline __tests__/__packages__/cli-test-utils/src/TestUtils.ts:139]: Curious if we should remove this reference to `IHandlerParameters.profiles` from V3? I don't want to risk breaking existing tests though 😋

**Review by t1m0thyj — APPROVED** [review]
LGTM, thanks @awharn for updating integration tests to run more stable on MacOS and porting my PR #2218 😋

**Review by traeok — APPROVED** [review]
LGTM, thanks Andrew

File: __tests__/__packages__/cli-test-utils/src/TestUtils.ts (hunk 1 above)
Comment (awharn) [reply __tests__/__packages__/cli-test-utils/src/TestUtils.ts:139]: I can take a look. I don't know if anything is still using it.

**Pull request merged**